mobile/src/services/mentors-api/mentors-api.service.ts

```diff
@@ -37,6 +39,20 @@ class MentorsApi {
       },
     );
   }
+
+  public getMenteesMentor({
+    courseId,
+    menteeId,
+  }: GetMentorRequestParamsDto): Promise<MenteesToMentorsResponseDto> {
+    return this.#http.load<MenteesToMentorsResponseDto>(
+      `${this.#apiPrefix}${ApiPath.MENTORS}${
+        MentorsApiPath.COURSES
+      }/${courseId}${MentorsApiPath.MENTEES}/${menteeId}`,
+      {
+        method: HttpMethod.GET,
+      },
```

Comment:
I think, not necessary to write method GET, we have that by default into http.service 
![image](https://user-images.githubusercontent.com/82529236/188492211-e37a4842-33c3-4b64-917f-346de362bf7c.png)